### PR TITLE
fix: github actions e2e workflow

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-100](https://klau.atlassian.net/browse/FSU-100)

## Describe your changes
This pull request addresses an issue with the previous e2e GitHub Actions configuration by switching to use a localhost URL instead of a Vercel deployment URL. The previous setup caused problems because it relied on two separate deployments - one for the design system and another for the frontend - and Cypress was only able to test against the first deployed app. This resulted in inconsistent test results and the potential for bugs to go undetected. This change improves the reliability and accuracy of the e2e testing process by ensuring that Cypress is always testing against the correct app.

☝️ description generated by [ChatGTP3](https://chat.openai.com/chat), just for fun 😂 

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by AmandaKr — COMMENTED** [review]
image
test don't pass 🤔

**Review by paulgrym — APPROVED** [review]
🚀

**Review by AmandaKr — APPROVED** [review]
🥇

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        uses: actions/checkout@v3
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v5
+        with:
+          build: yarn build
+          start: yarn start
+          wait-on: "http://localhost:3000"
+          browser: chrome
+          project: packages/frontend/e2e
```
Comment (im-klau) [inline .github/workflows/e2e.yml:17]:
```suggestion
          project: ./packages/frontend/e2e
```

**Review by Ebmili — APPROVED** [review]
Well done :)

**Pull request merged**